paper/paper.md

```diff
@@ -198,10 +205,174 @@ solver_stats = trunk(
     )
 
 final_accuracy = KnetNLPModels.accuracy(LeNetNLPModel)
+```
+## FluxNLPModels.jl
+We will define the neural network architecture using Flux.jl, specifically the LeNet architecture [@lecun1998gradient]. First, we define some methods to assist with calculations of accuracy, data loading, and training arguments such as the number of epochs and batch size. These methods are inspired by Flux-zoo [@flux_model_zoo].
+```julia
+using FluxNLPModels
+using CUDA, Flux, NLPModels
+using Statistics
+using Flux.Data: DataLoader
+using Flux: onehotbatch, onecold, @epochs
+using Flux.Losses: logitcrossentropy
+using Base: @kwdef
+using MLDatasets
+using JSOSolvers
+```
+
+
+#### Loading Data
+In this section, we will cover the process of loading datasets and defining minibatches for training your model using Flux. Loading and preprocessing data is an essential step in machine learning, as it allows you to train your model on real-world examples.
+
+We will specifically focus on loading the MNIST dataset. We will divide the data into training and testing sets, ensuring that we have separate data for model training and evaluation.
+
+Additionally, we will define minibatches, which are subsets of the dataset that are used during the training process. Minibatches enable efficient training by processing a small batch of examples at a time, instead of the entire dataset. This technique helps in managing memory resources and improving convergence speed.
+To donwnload and load MNIST dataset [@lecun-bouttou-bengio-haffner1998] from MLDataset:
+```julia
+
+function getdata(args; T = Float32) #T for types
+  ENV["DATADEPS_ALWAYS_ACCEPT"] = "true"
+
+  # Loading Dataset	
+  xtrain, ytrain = MLDatasets.MNIST(Tx = T, split = :train)[:]
+  xtest, ytest = MLDatasets.MNIST(Tx = T, split = :test)[:]
+
+  # Reshape Data in order to flatten each image into a linear array
+  xtrain = Flux.flatten(xtrain)
+  xtest = Flux.flatten(xtest)
+
+  # One-hot-encode the labels
+  ytrain, ytest = onehotbatch(ytrain, 0:9), onehotbatch(ytest, 0:9)
+
+  # Create DataLoaders (mini-batch iterators)
+  train_loader = DataLoader((xtrain, ytrain), batchsize = args.batchsize, shuffle = true)
+  test_loader = DataLoader((xtest, ytest), batchsize = args.batchsize)
+  @info "The type is " typeof(xtrain)
+  return train_loader, test_loader
+end
+```
```

Comment:
This should be common with the KnetNLPModels.jl example

Reply:
Sure, we can 
But here we create the data loader in the function as well 

I will change mine to be a bit different and @paraynaud  can you modify your code as well to use the function I make?

Reply:
I fixed it 
I will push this

Reply:
I will move and adapt the KnetNLPModel example when this PR is rebased.

Reply:
I edited the example